packages/apps/graph/src/devnet/deployment/helper.ts

```diff
@@ -0,0 +1,72 @@
+import type {
+  ChainId,
+  IClient,
+  ICommand,
+  ICommandResult,
+  IKeyPair,
+  ITransactionDescriptor,
+  IUnsignedCommand,
+} from '@kadena/client';
+import {
+  addSignatures,
+  createClient,
+  isSignedTransaction,
+} from '@kadena/client';
+import { sign } from '@kadena/cryptography-utils';
+import { dotenv } from '@utils/dotenv';
+import { logger } from '@utils/logger';
+
+export interface IAccount {
+  account: string;
+  chainId?: ChainId;
+  keys: IKeyPair[];
+}
+
+const getClient = (): IClient =>
+  createClient(
+    ({ chainId }) =>
+      `${dotenv.NETWORK_HOST}/chainweb/0.0/${dotenv.NETWORK_ID}/chain/${chainId}/pact`,
+  );
+
+export const submit = (tx: ICommand): Promise<ITransactionDescriptor> =>
+  getClient().submit(tx);
+
+export const listen = (tx: ITransactionDescriptor): Promise<ICommandResult> =>
+  getClient().listen(tx);
```

Comment:
Any particular reason to recreate the client every time you need it? You could use the same client by calling `getClient` once.

Reply:
Great catch. Changes made :)